api/net/Areas/Subscriber/Controllers/ContentController.cs

```diff
@@ -77,6 +77,10 @@ public async Task<IActionResult> FindWithElasticsearchAsync()
         var uri = new Uri(this.Request.GetDisplayUrl());
         var query = Microsoft.AspNetCore.WebUtilities.QueryHelpers.ParseQuery(uri.Query);
         var filter = new ContentFilter(query);
+        if (filter.Actions.Any(x => x.Contains("Commentary")))
```

Comment:
You should check if Quantity has been set so that it uses the correct value.

Reply:
Updated, thanks